frontend/plan/components/search/SearchBar.tsx

```diff
@@ -5,6 +5,8 @@ import styled from "styled-components";
 // @ts-ignore
 import AccountIndicator from "pcx-shared-components/src/accounts/AccountIndicator";
 import { useRouter } from "next/router";
+import * as ReactDOM from "react-dom";
+import dynamic from "next/dynamic";
```

Comment:
Are these unused imports? If so, can you remove them

Reply:
Yup, good catch.